includes/cf-descriptions/pathe.md

```diff
@@ -1,3 +1,5 @@
 **Pathé Thuis**<br>
 
 [From Wikipedia, the free encyclopedia](https://nl.wikipedia.org/wiki/Path%C3%A9_Thuis){:target="_blank" rel="noopener noreferrer"}
+
+Pathé Thuis is a Dutch streaming services, that you pay per movie you want to watch.
```

Comment:
```suggestion
[From Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Path%C3%A9_Thuis){:target="_blank" rel="noopener noreferrer"}

Pathé Thuis is a pay-per-movie Dutch streaming service.

Reply:
So I just double checked the link, there is no en. version of this streaming service on Wikipedia. Weird.

Not sure if you want to cite Wikipedia in dutch? Not sure it really matters though. Probably not a widely used link.